Description: Fix path to use default build. Will be applied upstream ASAP
Author: Jacob Adams <tookmund@gmail.com>

Index: swapspace-1.11-1/swapspace.service
===================================================================
--- swapspace-1.11-1.orig/swapspace.service
+++ swapspace-1.11-1/swapspace.service
@@ -7,7 +7,7 @@ Requires=swap.target
 
 [Service]
 Type=simple
-ExecStart=/sbin/swapspace
+ExecStart=/usr/sbin/swapspace
 Restart=always
 RestartSec=30
 
